frontend/src/components/PropertyForm.js

```diff
@@ -1,20 +1,38 @@
 import React from "react";
-import { Button, Form } from "react-bootstrap";
+import { Button, Form, Image } from 'react-bootstrap'
+
+import AddressImage from "../images/address.svg"
+import DateImage from "../images/date.svg"
+import DollarsignImage from "../images/dollarsign.svg"
+import HouseImage from "../images/house.svg"
+import BedImage from "../images/bed.svg"
+import BathImage from "../images/bath.svg"
+import SchoolImage from "../images/school.svg"
+import TicketImage from "../images/ticket.svg"
+import PersonImage from "../images/person.svg"
+import PhoneImage from "../images/phone.svg"
+import LocationImage from "../images/location.png"
+import BusImage from "../images/bus.svg"
+import BasementImage from "../images/basement.svg"
```

Comment:
add an index.js file to the images directory. Then do:
```suggestion
import * as images from "../images"
```
and access like so:
```js
images.BusImage
```

Reply:
I implemented an index file, but I only imported the ones we were using.  Let me know if this is ok.